packages/driver/src/cypress/server.coffee

```diff
@@ -31,7 +31,9 @@ nope = -> return null
 isAbortedThroughUnload = (xhr) ->
   xhr.readyState is 4 and
     xhr.status is 0 and
-      xhr.responseText is ""
+      ## TODO: How do we want to handle other responseTypes?
+      (xhr.responseType is "" or xhr.responseType is "text") and
```

Comment:
- let's leave a comment here pointing to the issue that describes why `''` and `'text'` are the only two responseTypes we are looking at
- let's wrap this check up into a method that describes these two values: something like `responseTypeIsTextOrBlank`